spec/features/partner_edit_feature_spec.rb

```diff
@@ -92,7 +92,6 @@
 
     click_button "Update Information"
 
-    expect(page).to have_content "pending"
```

Comment:
Im curious about this removal. I _think_ that we still want this assertion? Ill take a look.